pkg/quota/evaluator/core/services.go

```diff
@@ -37,9 +41,31 @@ func NewServiceEvaluator(kubeClient clientset.Interface) quota.Evaluator {
 		MatchedResourceNames: allResources,
 		MatchesScopeFunc:     generic.MatchesNoScopeFunc,
 		ConstraintsFunc:      generic.ObjectCountConstraintsFunc(api.ResourceServices),
-		UsageFunc:            generic.ObjectCountUsageFunc(api.ResourceServices),
+		UsageFunc:            ServiceUsageFunc,
 		ListFuncByNamespace: func(namespace string, options api.ListOptions) (runtime.Object, error) {
 			return kubeClient.Core().Services(namespace).List(options)
 		},
 	}
 }
+
+// ServiceUsageFunc knows how to measure usage associated with services
+func ServiceUsageFunc(object runtime.Object) api.ResourceList {
+	result := api.ResourceList{}
+	if service, ok := object.(*api.Service); ok {
+		result[api.ResourceServices] = resource.MustParse("1")
+		switch service.Spec.Type {
+		case api.ServiceTypeNodePort:
+			result[api.ResourceServicesNodePorts] = resource.MustParse("1")
+		}
+	}
+	return result
+}
+
+// QuotaService returns true if the service is eligible to track against a quota
+func QuotaService(service *api.Service) bool {
```

Comment:
I think we should rename this since a service is always eligible to track against a quota if the quota is just on 'services' and not 'services.nodeports'.  Can you rename this `QuotaServiceType` returns true if the service type is eligible to track against a quota?